app/styles/components/_form_styles.scss

```diff
@@ -6,3 +6,13 @@
 
   select { height: 44px; }
 }
+
+.text-expansion-div {
```

Comment:
To keep classnames clean, lets change this to just `.text-expansion` or `text-expansion-container`.